specification/sdk-tracing.md

```diff
@@ -110,8 +116,18 @@ invocations. The span processors are invoked only when
 must be used to implement [span exporter](#span-exporter) to batch and convert
 spans.
 
-Span processors can be registered directly on SDK Tracer and they are invoked in
-the same order as they were registered.
+Span processors can be registered directly on SDK `TracerFactory` and they are
+invoked in the same order as they were registered.
+
+All `Tracer` instances created by a `TracerFactory` share the same span processors.
+Changes to this collection reflect in all `Tracer` instances.
+Implementation-wise, this could mean that `Tracer` instances have a reference to
+their `TracerFactory` and can access span processor objects only via this
```

Comment:
Above it's mentioned that `Tracers` are created via the `TraceFactory` but reading this here sounds like we could have more then one `TraceFactory`. If there are more factories, how to get that one I should use in instrumentation code to create a tracer?

Reply:
By default there is a global instance of `TracerFactory` available, but there could be more than one `TracerFactory`.
It's basically the same way as it was for `Tracer` instances before. There was a global instance but you could always `new Tracer`.

Reply:
So we may get a `TracerFactoryFactory`? Hopefully that one will never have more then one instance... 😁

Reply:
On an API level, there will be the global "TracerFactory" (the comment above was meant for SDKs)